api/nakadi-event-bus-api.yaml

```diff
@@ -0,0 +1,1218 @@
+          rejected if any of the rules fail (see details of Problem response on the Event publishing
+          methods).
+
+          Rule evaluation order is the same as in this array.
+
+          If not explicitly set, default value will respect the definition of the `EventType.type`.
+        type: array
+        items:
+          $ref: '#/definitions/EventValidationStrategy'
+
+      enrichment_strategies:
+        description: |
+          Determines the enrichment to be performed on an Event upon reception. Enrichment is performed once
+          upon reception (and after validation) of an Event and is only possible on fields that are not
+          defined on the incoming Event.
+
+          See documentation for the write operation for details on behaviour in case of unsuccessful
+          enrichment.
+        type: array
+        items:
+          $ref: '#/definitions/EventEnrichmentStrategy'
+        default: [{name: 'metadata-enrichment'}]
+
+
+      partition_resolution_strategy:
+        description: |
+          Determines how the assignment of the event to a Partition should be handled.
+
+        type: object
+        $ref: '#/definitions/PartitionResolutionStrategy'
+
+      schema:
+        type: object
+        $ref: '#/definitions/EventTypeSchema'
+        description: |
+          The schema for this EventType. This is expected to be a json schema in yaml format (other formats
+          might be added in the future).
+
+      data_key:
+        type: array
+        items:
+          type: string
+        description: |
+          Indicators of the path of the properties that constitute the primary key (identifier) of the data
+          within this Event.
+          If set MUST be a valid required field as defined in the schema.
+          (TBD should be required? Is applicable only to both Business and DataChange Events?)
+
+      ordering_key:
+        type: string
+        description: |
+          Indicator of the field used for guaranteeing the ordering of Events of this type (used by the
+          PartitionResolutionStrategy).
+          If set MUST be a valid required field as defined in the schema.
+
+    required:
+      - name
+
+  EventTypeSchema:
+    properties:
+      type:
+        type: string
+        example: 'json-schema'
+        description: The type of schema definition (avro, json-schema, etc).
+      schema:
+        type: string
+        description: |
+          The schema as string in the syntax defined in the field type. Failure to respect the syntax will
+          fail any operation on an EventType.
+    required:
+      - type
+      - schema
+
+  EventValidationStrategy:
+    description: |
+      Defines a rule for validation of an incoming Event.
+
+      Rules might require additional parameters; see the `doc` field of the existing rules for details. See
+      GET /registry/validation-rules for a list of available rules.
```

Comment:
The actual endpoint is `GET /registry/validation-rule` not `GET /registry/validation-rules`